### ISSUE TITLE
Error in unified api getting the ADPs

### ISSUE THREAD

**Issue by rosogon** [SeaCloudsEU/SeaCloudsPlatform#165]
@adriannieto : The planner successfully return a set of adps, but the dashboard raises an exception:

```
ERROR [2015-11-27 13:24:55,513] io.dropwizard.jersey.errors.LoggingExceptionMapper: Error handling a request: db8797eaadf9d9d2
! java.lang.IllegalStateException: null
! at com.google.gson.JsonArray.getAsString(JsonArray.java:179) ~[dashboard.jar:na]
! at eu.seaclouds.platform.dashboard.resources.PlannerResource.getAdps(PlannerResource.java:71) ~[dashboard.jar:na]
! at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method) ~[na:1.8.0_31]
! at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source) ~[na:1.8.0_31]
! at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source) ~[na:1.8.0_31]
! at java.lang.reflect.Method.invoke(Unknown Source) ~[na:1.8.0_31]
! at org.glassfish.jersey.server.model.internal.ResourceMethodInvocationHandlerFactory$1.invoke(ResourceMethodInvocationHandlerFactory.java:81) ~[dashboard.jar:na]
! at org.glassfish.jersey.server.model.internal.AbstractJavaResourceMethodDispatcher$1.run(AbstractJavaResourceMethodDispatcher.java:164) ~[dashboard.jar:na]
! at org.glassfish.jersey.server.model.internal.AbstractJavaResourceMethodDispatcher.invoke(AbstractJavaResourceMethodDispatcher.java:181) ~[dashboard.jar:na]
! at org.glassfish.jersey.server.model.internal.JavaResourceMethodDispatcherProvider$ResponseOutInvoker.doDispatch(JavaResourceMethodDispatcherProvider.java:158) ~[dashboard.jar:na]
! at org.glassfish.jersey.server.model.internal.AbstractJavaResourceMethodDispatcher.dispatch(AbstractJavaResourceMethodDispatcher.java:101) ~[dashboard.jar:na]
! at org.glassfish.jersey.server.model.ResourceMethodInvoker.invoke(ResourceMethodInvoker.java:389) ~[dashboard.jar:na]
! at org.glassfish.jersey.server.model.ResourceMethodInvoker.apply(ResourceMethodInvoker.java:347) ~[dashboard.jar:na]
! at org.glassfish.jersey.server.model.ResourceMethodInvoker.apply(ResourceMethodInvoker.java:102) ~[dashboard.jar:na]
! at org.glassfish.jersey.server.ServerRuntime$2.run(ServerRuntime.java:305) ~[dashboard.jar:na]
! at org.glassfish.jersey.internal.Errors$1.call(Errors.java:271) [dashboard.jar:na]
! at org.glassfish.jersey.internal.Errors$1.call(Errors.java:267) [dashboard.jar:na]
! at org.glassfish.jersey.internal.Errors.process(Errors.java:315) [dashboard.jar:na]
! at org.glassfish.jersey.internal.Errors.process(Errors.java:297) [dashboard.jar:na]
! at org.glassfish.jersey.internal.Errors.process(Errors.java:267) [dashboard.jar:na]
! at org.glassfish.jersey.process.internal.RequestScope.runInScope(RequestScope.java:317) [dashboard.jar:na]
! at org.glassfish.jersey.server.ServerRuntime.process(ServerRuntime.java:288) [dashboard.jar:na]
! at org.glassfish.jersey.server.ApplicationHandler.handle(ApplicationHandler.java:1110) [dashboard.jar:na]
! at org.glassfish.jersey.servlet.WebComponent.service(WebComponent.java:401) [dashboard.jar:na]
! at org.glassfish.jersey.servlet.ServletContainer.service(ServletContainer.java:386) [dashboard.jar:na]
! at org.glassfish.jersey.servlet.ServletContainer.service(ServletContainer.java:335) [dashboard.jar:na]
! at org.glassfish.jersey.servlet.ServletContainer.service(ServletContainer.java:222) [dashboard.jar:na]
! at io.dropwizard.jetty.NonblockingServletHolder.handle(NonblockingServletHolder.java:49) [dashboard.jar:na]
! at org.eclipse.jetty.servlet.ServletHandler$CachedChain.doFilter(ServletHandler.java:1669) [dashboard.jar:na]

! at org.eclipse.jetty.servlets.UserAgentFilter.doFilter(UserAgentFilter.java:83) [dashboard.jar:na]
! at org.eclipse.jetty.servlets.GzipFilter.doFilter(GzipFilter.java:300) [dashboard.jar:na]
! at io.dropwizard.jetty.BiDiGzipFilter.doFilter(BiDiGzipFilter.java:134) [dashboard.jar:na]
! at org.eclipse.jetty.servlet.ServletHandler$CachedChain.doFilter(ServletHandler.java:1652) [dashboard.jar:na]

! at io.dropwizard.servlets.ThreadNameFilter.doFilter(ThreadNameFilter.java:29) [dashboard.jar:na]
! at org.eclipse.jetty.servlet.ServletHandler$CachedChain.doFilter(ServletHandler.java:1652) [dashboard.jar:na]

! at io.dropwizard.jersey.filter.AllowedMethodsFilter.handle(AllowedMethodsFilter.java:44) [dashboard.jar:na]
! at io.dropwizard.jersey.filter.AllowedMethodsFilter.doFilter(AllowedMethodsFilter.java:39) [dashboard.jar:na]

! at org.eclipse.jetty.servlet.ServletHandler$CachedChain.doFilter(ServletHandler.java:1652) [dashboard.jar:na]

! at org.eclipse.jetty.servlet.ServletHandler.doHandle(ServletHandler.java:585) [dashboard.jar:na]
! at org.eclipse.jetty.server.handler.ContextHandler.doHandle(ContextHandler.java:1127) [dashboard.jar:na]
! at org.eclipse.jetty.servlet.ServletHandler.doScope(ServletHandler.java:515) [dashboard.jar:na]
! at org.eclipse.jetty.server.handler.ContextHandler.doScope(ContextHandler.java:1061) [dashboard.jar:na]
! at org.eclipse.jetty.server.handler.ScopedHandler.handle(ScopedHandler.java:141) [dashboard.jar:na]
! at org.eclipse.jetty.server.handler.HandlerWrapper.handle(HandlerWrapper.java:97) [dashboard.jar:na]
! at com.codahale.metrics.jetty9.InstrumentedHandler.handle(InstrumentedHandler.java:240) [dashboard.jar:na]
! at io.dropwizard.jetty.ContextRoutingHandler.handle(ContextRoutingHandler.java:38) [dashboard.jar:na]
! at org.eclipse.jetty.server.handler.HandlerWrapper.handle(HandlerWrapper.java:97) [dashboard.jar:na]
! at org.eclipse.jetty.server.handler.RequestLogHandler.handle(RequestLogHandler.java:95) [dashboard.jar:na]
! at org.eclipse.jetty.server.handler.HandlerWrapper.handle(HandlerWrapper.java:97) [dashboard.jar:na]
! at org.eclipse.jetty.server.handler.StatisticsHandler.handle(StatisticsHandler.java:159) [dashboard.jar:na]
! at org.eclipse.jetty.server.handler.HandlerWrapper.handle(HandlerWrapper.java:97) [dashboard.jar:na]
! at org.eclipse.jetty.server.Server.handle(Server.java:497) [dashboard.jar:na]
! at org.eclipse.jetty.server.HttpChannel.handle(HttpChannel.java:310) [dashboard.jar:na]
! at org.eclipse.jetty.server.HttpConnection.onFillable(HttpConnection.java:257) [dashboard.jar:na]
! at org.eclipse.jetty.io.AbstractConnection$2.run(AbstractConnection.java:540) [dashboard.jar:na]
! at org.eclipse.jetty.util.thread.QueuedThreadPool.runJob(QueuedThreadPool.java:635) [dashboard.jar:na]
! at org.eclipse.jetty.util.thread.QueuedThreadPool$3.run(QueuedThreadPool.java:555) [dashboard.jar:na]
! at java.lang.Thread.run(Unknown Source) [na:1.8.0_31]
```

Please, coordinate with @szenzaro: I think the planner output has changed.

**Comment by adriannieto** [comment]
That was an expected problem, now I only have to foward the plan instead of parsing it. I will fixit ASAP.

**Comment by rosogon** [comment]
Fixed with #166
